buildSrc/src/main/groovy/io/micronaut/guides/IndexGenerator.groovy

```diff
@@ -31,52 +36,99 @@ class IndexGenerator {
-            save(templateText, metadata.slug + '.html', distDir, [metadata], metadata.title)
+            save(templateText, metadata.slug + '.html', distDir, [new GuidesSection(category: metadata.category, metadatas: [metadata])],  metadata.title)
         }
     }
 
     private static void save(String templateText,
                              String filename,
                              File distDir,
-                             List<GuideMetadata> metadatas,
-                             String title = 'Micronaut Guides') {
-        String text = indexText(distDir, templateText, metadatas, title)
+                             List<GuidesSection> sections,
+                             String title,
+                             Collection<Tag> tags = []) {
+        String text = indexText(distDir, templateText, sections, tags, title)
         File output = new File(distDir, filename)
         output.createNewFile()
         output.setText(text, 'UTF-8')
     }
 
     private static String indexText(File distDir,
                                     String templateText,
-                                    List<GuideMetadata> metadatas,
+                                    List<GuidesSection> sections,
+                                    Collection<Tag> tags,
                                     String title) {
-        boolean singleGuide = metadatas.size() == 1
+        boolean singleGuide = sections && sections.size() == 1 && sections.get(0).metadatas.size() == 1
+        List<GuideMetadata> metadatas = []
+        for (GuidesSection section : sections) {
+            metadatas.addAll(section.metadatas)
+        }
 
         String baseURL = System.getenv("CI") ? LATEST_GUIDES_URL : ""
         String index = ''
+        if (!singleGuide && tags) {
+            index += '<div class="categorygrid" style="padding-bottom: 20px;margin-bottom: 0;border-bottom: 0;">'
+            index += '<div class="grid">'
+            index += '  <div class="grid-item grid-item_primary grid-item_one-third grid-item_dynamic-height">'
+            index += '    <div class="inner">'
+            index += '      <h1 class="title title_large first-word-bold first-word-break"><strong>Micronaut</strong> Guides</h1>'
+            index += '    </div>'
+            index += '  </div>'
+            index += '  <div class="grid-item grid-item_white grid-item_two-third grid-item_dynamic-height">'
+            index += '    <div class="inner" style="padding: 0">'
+            index += guidesTable(latestGuides(metadatas), "Latest Guides", true)
+            index += '    </div>'
+            index += '  </div>'
+            index += '</div>'
+            index += '</div>'
+            index += '<div class="categorygrid">'
+            index += '<div class="row">'
+            index += '    <div class="inner" style="padding: 0">'
+            index += TagCloud.tagCloud(tags)
+            index += '    </div>'
+            index += '</div>'
```

Comment:
Think we need a new class on this, Row has

```
margin-right: -15px;
margin-left: -15px;
```

Which causes the tags to go outside of the DIV border

![image](https://user-images.githubusercontent.com/49317/174333457-711bf450-2d6f-4971-90ca-b0f6a1ffdbe6.png)